Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 gem 'aws-sdk'
 gem 'carrierwave', '~> 0.10.0'
 gem 'carrierwave-mongoid', '~> 0.8.1', require: 'carrierwave/mongoid'
-gem 'gds-sso', '~> 11.2'
+gem 'gds-sso', '~> 13.2'
```

Comment:
The [`CHANGELOG` for `gds-sso` v13.0.0](https://github.com/alphagov/gds-sso/blob/9478045b27fd2aa9b2832c7d3df266eea43dab47/CHANGELOG.md#1300) mentions two breaking changes. Can you confirm that these do not affect this app?